packages/api/src/db/comment.ts

```diff
@@ -106,6 +106,11 @@ export interface GetCommentsArgs {
   readonly order: SortOrder
 }
 
+export interface GetPublicCommentsArgs {
+  readonly id: string
+  readonly userID: string
```

Comment:
`userID` should be optional right? because you don't always have the user

Reply:
My bad